lib/migrator.js

```diff
@@ -119,6 +119,40 @@ Migrator.prototype = {
     }
   },
 
+  check: function (funcOrOpts, callback) {
```

Comment:
up and down was implicitly covered by the driver tests, which have been moved together with their drivers. So you're right, that these need to be created as well, not by you though :). However, if you could add a unit test for this function, validating that it returns the expected result for a given input. You can mock away `loadFromFileSystem` and `loadFromDatabase`, those are already proven to work.